hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/scheduler/HdfsScheduler.java

```diff
@@ -199,6 +202,10 @@ public void statusUpdate(SchedulerDriver driver, TaskStatus status) {
   public void resourceOffers(SchedulerDriver driver, List<Offer> offers) {
     log.info(String.format("Received %d offers", offers.size()));
 
```

Comment:
Let's check liveState.getCurrentAcquisitionPhase() == AcquisitionPhase.RECONCILING_TASKS before reconciler.complete. Swap the values in your && statement. This is a nit, but it's nice to not enter logic unless we must.